libraries/plugins/es_objects/es_objects.cpp

```diff
@@ -260,6 +316,7 @@ void es_objects_plugin::plugin_set_program_options(
          ("es-objects-index-prefix", boost::program_options::value<std::string>(), "Add a prefix to the index(objects-)")
          ("es-objects-keep-only-current", boost::program_options::value<bool>(), "Keep only current state of the objects(true)")
          ("es-objects-start-es-after-block", boost::program_options::value<uint32_t>(), "Start doing ES job after block(0)")
+         ("es-objects-load-genesis", boost::program_options::value<bool>(), "Load genesis data to ES(true)")
```

Comment:
The documentation string is misleading. The option will index whatever is in the database on startup.
Also, `genesis()` is called after the first change, which includes changes that were applied after genesis.